### ISSUE TITLE
LIBAVDEVICE not found

### ISSUE THREAD

**Issue by kjcole** [qTox/qTox#4283]
##### Brief Description

**OS:** Ubuntu Studio Linux 16.10 (yackkety)

```
$ lsb_release --all
LSB Version:	core-9.20160110ubuntu5-amd64:core-9.20160110ubuntu5-noarch:printing-9.20160110ubuntu5-amd64:printing-9.20160110ubuntu5-noarch:security-9.20160110ubuntu5-amd64:security-9.20160110ubuntu5-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 16.10
Release:	16.10
Codename:	yakkety
```

**qTox version:** unavailable. compile fails.
**Commit hash:**  e9515f4898aedfd9c70094d61a6f3b85debb9e3e
```
$ git log
commit e9515f4898aedfd9c70094d61a6f3b85debb9e3e
Author: sudden6 <sudden6@gmx.at>
Date:   Tue Mar 21 19:40:27 2017 +0100

    fix: remove useless line that caused a warning
```  

**toxcore:**  bbdd7982563b7d8f89fcd3b7297b5fbdf21fc57f
```
$ git log
commit bbdd7982563b7d8f89fcd3b7297b5fbdf21fc57f
Author: iphydf <iphydf@users.noreply.github.com>
Date:   Wed Dec 14 09:17:19 2016 +0000

    Fix unresolved reference in toxencryptsave API.

    Also, make sure this won't happen again by checking for it in
    format-source.
```
**Qt:**  5.6.1

**Hardware:** Acer C710 Chromebook (rooted, SeaBIOS, John Lewis's JELTKA)

**Reproducible:** Always

##### Steps to reproduce

1. cmake .

##### Observed Behavior

```
$ cmake .
-- Checking for one of the modules 'libavdevice'
CMake Error at cmake/Dependencies.cmake:72 (message):
  LIBAVDEVICE package, library or framework not found
Call Stack (most recent call first):
  cmake/Dependencies.cmake:85 (search_dependency)
  CMakeLists.txt:73 (include)

-- Configuring incomplete, errors occurred!
```

##### Expected Behavior

No errors.

##### Additional Info

```
$ dpkg -l | grep libavdevice
ii  libavdevice-ffmpeg56:amd64                                  7:2.8.6-1ubuntu2                            amd64        FFmpeg library for handling input and output devices - runtime files
ii  libavdevice57:amd64                                         7:3.0.7-0ubuntu0.16.10.1                    amd64        FFmpeg library for handling input and output devices - runtime files
```

**Comment by zetok** [comment]
Have you tried to install required deps, listed in INSTALL.md?

github changed their markdown parser, and current version of INSTALL.md on master might be in places hard to parse; corrected version is in #4282

**Comment by kjcole** [comment]
I'm feeling old. That's twice today that I read install docs and missed something. In this case, I installed the Debian dependencies instead of the Ubuntu dependencies. (In fairness, the stanzas looked fairly close to each other.  But, the difference turned out to be three packages. My bad.
